packages/ds-medicare-gov/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@cmsgov/ds-medicare-gov",
-  "version": "3.4.0-beta.2",
+  "version": "3.5.0-beta.1",
```

Comment:
I should have mentioned this first, but for the medicare one, we're just incrementing the beta version number for now until PI-1

Reply:
Ok, am I correct in thinking I can just update that here, remove that particular tag then add the new tag? Just want to be sure there isn't anything I'm missing.

Reply:
Unfortunately I think it would mean redoing all the tags, because the tags should all point to the same commit even though they have different names.